serviceWorker.js

```diff
@@ -89,3 +96,27 @@ registerRoute(
 );
 
 workbox.precaching.precacheAndRoute([{url: 'index.html', revision: '1.0.6'}]);
```

Comment:
Is peaching needed for index.html?

Reply:
I suppose not. Is this because we don't need to do a clean up of the old cache from the activate listener below?